README.md

```diff
@@ -157,3 +157,50 @@ Common usage:
   ```shell
   docker run --rm -v ~/solid-config:/config -p 3000:3000 -it css:latest -c /config/my-config.json
   ```
+
+## Using the identity provider
+
+1. Launch the Community Solid Server, for example in file server mode:
+    ```bash
+    git clone git@github.com:solid/community-server.git
+    cd community-server
+    mkdir data
+    npm ci
+    npm run start -- -c config/config-file.json -f ./data
+    ```
+2. To use the identity provider, you need a compatible client application.
+
+    You can use for example `@inrupt/solid-client-authn-js`:
+
+    ```bash
+    git clone https://github.com/inrupt/solid-client-authn-js
+    cd solid-client-authn-js
+    npm ci
+    cd packages/node/example/demoClientApp/
+    npm ci
+    npm run start
```

Comment:
Fun fact: `npm start` also works (as it's one of the default scripts npm recognizes).

Reply:
Haha, true ^^,